modules/sdk-core/src/bitgo/wallet/wallets.ts

```diff
@@ -463,6 +463,14 @@ export class Wallets implements IWallets {
       walletParams.rootPrivateKey = params.rootPrivateKey;
     }
 
+    // Custodial onchain wallets do not need m, n, keys, or keySignatures
+    if (params.type === 'custodial' && params.multisigType === 'onchain') {
+      walletParams.n = undefined;
+      walletParams.m = undefined;
+      walletParams.keys = [];
```

Comment:
```suggestion
      walletParams.keys = undefined;
```
maybe

Reply:
I decided to use `keys = []` because it meant I wouldn't need to make the `keys` field optional in `iBaseCoin`. But I think keeping things consistent here is ideal, so I wouldn't mind making this change.